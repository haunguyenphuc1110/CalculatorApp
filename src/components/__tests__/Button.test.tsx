import React from 'react';
import renderer from 'react-test-renderer';

import Button from '../Button.component';

it('renders correctly', () => {
  const props = {
    value: '9',
    onPress: jest.fn(),
  };
  const tree = renderer.create(<Button {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
