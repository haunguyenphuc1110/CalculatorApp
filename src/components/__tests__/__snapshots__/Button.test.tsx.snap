// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "#921029",
        "borderWidth": 1,
        "flex": 1,
        "justifyContent": "center",
      },
      null,
    ]
  }
  testID="button[9]"
>
  <Text
    style={
      Object {
        "color": "#000000",
        "fontSize": 30,
        "fontWeight": "bold",
      }
    }
  >
    9
  </Text>
</View>
`;
